api/graphql/schema.graphql

```diff
@@ -1225,7 +1225,7 @@ input UpdateUserAsAdminInput
   educationExperiences: EducationExperienceHasMany
   awardExperiences: AwardExperienceHasMany
 
-  roles: RoleAssignmentHasMany
+  rolesSetter: RoleAssignmentHasMany
```

Comment:
Hmm, yeah this is a bit awkward.  It has bothered me ever since I set this up that this property was not called `roleAssignments` like it is on the User type.  Could you try that?  Or if you have to go with a non-standard method on the model could you use the rename directive to give it a nice name in the schema?

Reply:
done in https://github.com/GCTC-NTGC/gc-digital-talent/pull/6291/commits/32dcf653157d2fc3572e45ea2aa50a2d038f92c5